### DISCUSSION TITLE
Set Window Title to Match Loaded Workflow?

### DISCUSSION THREAD

**Discussion by BlastedRemnants** [comfyanonymous/ComfyUI#4293] [Q&A] [answered]
Hi folks, I tried searching around a bit and asked CoPilot (which has been surprisingly helpful for other problems but not this one) and have had no luck so far, is there a way to have my browser tab name change to match a workflow when I load one? I usually run 2 or 3 tabs with different workflows for trying things and experimenting, but sometimes things can get a bit messy and when it comes time to update my saved workflow I don't always remember which tab is which.

I thought it'd be super handy to have the flow name appear as the tab title, but I have no idea how to make that happen. Anybody have any ideas? CoPilot suggested trying with AutoHotkey but I've had struggles using AHK for stuff before and really don't want to try that again, if I can avoid it. Thanks!

**Comment by ltdrdata** [comment, marked as answer]
@huchenlei It would be good if this feature is reflected in the new front.

**Reply by BlastedRemnants** [reply]
Oh right, I was just reading about that yesterday but forgot about it today apparently, oops! Would probably make more sense to wait for the new front end before trying to figure this out.
